assets/src/edit-story/components/panels/backgroundSizePosition.js

```diff
@@ -28,49 +27,57 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+import { Button, Row } from '../form';
 import useStory from '../../app/story/useStory';
-import { Button } from '../form';
 import { SimplePanel } from './panel';
+import getCommonValue from './utils/getCommonValue';
+import FlipControls from './shared/flipControls';
+
+function BackgroundSizePositionPanel({ selectedElements, onSetProperties }) {
+  const flip = getCommonValue(selectedElements, 'flip');
```

Comment:
Given the current implementation of the `getCommonValue`, no two components will have a common value, since it requires `===` equality. I think somewhere we did something like this:

```
const flip = {
  horizontal: getCommonValue(selectedElements.map({flip} => flip || {horizontal: false}), 'horizontal'),
  vertical: getCommonValue(selectedElements.map({flip} => flip || {vertical: false}), 'vartical'),
}
```

It might be a good time to make it an utility as well... Maybe a version of `getCommonValue` could accept a mapper function. E.g. `getCommonValue(selectedElements, ({flip}) => flip?.horizontal||false)`?

Reply:
Added `getCommonObjectValue` util.